webapp/src/components/admin_console_settings/rtcd_service_url/modals.tsx

```diff
@@ -65,6 +72,15 @@ export const OnPremTrialSuccess = (modalProps: Props) => {
                                 {text}
                             </a>
                         ),
+                        recordingsDocsLink: (text: string) => (
+                            <a
+                                href='https://docs.mattermost.com/configure/calls-recordings.html'
```

Comment:
```suggestion
                                href='https://docs.mattermost.com/configure/plugins-configuration-settings.html#enable-call-recording'
```

Reply:
@streamer45, there will also be content in the general documentation about call recording but this is where all the other settings are documented so it seemed like the best fit.

Reply:
Okay, as long as we link to the other content as there's going to be something dedicated to deploying the new job service for calls which is what really enables recordings, and a requirement for the feature.

Reply:
Ah, if that'll be added to that page that makes sense. I've updated my proposal.